### DISCUSSION TITLE
Shelly 2.5 Sensor Update Time

### DISCUSSION THREAD

**Discussion by Papablaubaer** [arendst/Tasmota#15641] [Q&A] [answered]
Is there a way to set the time after which the energy sensors of a Shelly 2.5 are read? I would like to send all sensor data via websend per rule, but now they are updated about every second, which is a bit too much.

**Comment by barbudor** [comment, marked as answer]
You can use a rule prefixed by `tele-` so they would be sent by your rule at the same time as the Teleperiod
which you can control the Teleperiod using the command of the same name

**Reply by sfromis** [reply]
If your purpose is to keep track of current energy consumption, without too much delay, I'd very much suggest using [PowerDelta](https://tasmota.github.io/docs/Commands/#powerdelta), to have Tasmota report new values after a threshold of change was seen. As as starting point, I usually set something like `PowerDelta 110` to have it react to a 10W change.
